publisher-html/src/main/resources/templates/events/event.hbs

```diff
@@ -1,14 +1,21 @@
 <div class="event">
-    <img class="event-image" src="/img/image_placeholder.svg"/>
+    <svg class="event-image" aria-hidden="true" viewBox="0 0 512 512" >
+        <use xlink:href="#image"></use>  
```

Comment:
instead of rewritten <svg> and <use> could extra to "small",  "medium", "large" handlebar partials